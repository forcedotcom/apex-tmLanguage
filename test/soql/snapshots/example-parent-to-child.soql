SELECT Name, (SELECT LastName FROM Contacts) FROM Account;

SELECT Account.Name, (SELECT Contact.LastName FROM Account.Contacts) FROM Account
