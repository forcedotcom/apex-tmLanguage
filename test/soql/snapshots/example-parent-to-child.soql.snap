>SELECT Name, (SELECT LastName FROM Contacts) FROM Account;
#^^^^^^ source.soql keyword.operator.query.select.apex
#      ^ source.soql
#       ^^^^ source.soql keyword.query.field.apex
#           ^ source.soql punctuation.separator.comma.apex
#            ^ source.soql
#             ^ source.soql punctuation.parenthesis.open.apex
#              ^^^^^^ source.soql keyword.operator.query.select.apex
#                    ^ source.soql
#                     ^^^^^^^^ source.soql keyword.query.field.apex
#                             ^ source.soql
#                              ^^^^ source.soql keyword.operator.query.from.apex
#                                  ^ source.soql
#                                   ^^^^^^^^ source.soql storage.type.apex
#                                           ^ source.soql punctuation.parenthesis.close.apex
#                                            ^ source.soql
#                                             ^^^^ source.soql keyword.operator.query.from.apex
#                                                 ^ source.soql
#                                                  ^^^^^^^ source.soql storage.type.apex
#                                                         ^^ source.soql
>
>SELECT Account.Name, (SELECT Contact.LastName FROM Account.Contacts) FROM Account
#^^^^^^ source.soql keyword.operator.query.select.apex
#      ^ source.soql
#       ^^^^^^^^^^^^ source.soql keyword.query.field.apex
#                   ^ source.soql punctuation.separator.comma.apex
#                    ^ source.soql
#                     ^ source.soql punctuation.parenthesis.open.apex
#                      ^^^^^^ source.soql keyword.operator.query.select.apex
#                            ^ source.soql
#                             ^^^^^^^^^^^^^^^^ source.soql keyword.query.field.apex
#                                             ^ source.soql
#                                              ^^^^ source.soql keyword.operator.query.from.apex
#                                                  ^ source.soql
#                                                   ^^^^^^^^^^^^^^^^ source.soql storage.type.apex
#                                                                   ^ source.soql punctuation.parenthesis.close.apex
#                                                                    ^ source.soql
#                                                                     ^^^^ source.soql keyword.operator.query.from.apex
#                                                                         ^ source.soql
#                                                                          ^^^^^^^ source.soql storage.type.apex
>